tests/cases/fourslash/refactorExtractType73.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: a.ts
+////interface Foo<T extends { prop: T }> {}
+
+// @Filename: b.ts
+////interface Foo<T extends { prop: /*a*/T/*b*/ }> {}
+
+goTo.file("b.ts");
+goTo.select("a", "b");
+verify.not.refactorAvailable("Extract type", "Extract to type alias");

```

Comment:
Is the reason that that test works because the file contents are identical? Consider this test case.

```ts
/// <reference path="./fourslash.ts" />

// @Filename: a.ts
///// interface Foo<T extends { prop: [|T|] }> {}

// @Filename: b.ts
//// // Some initial comments.
//// // We need to ensure these files have different contents,
//// // so their ranges differ, so we'll start a few lines below in this file.
//// interface Foo<T extends { prop: [|T|] }> {}

for (const range of test.ranges()) {
    goTo.selectRange(range);
    verify.not.refactorAvailable("Extract type", "Extract to type alias");
}
```